python/vtool/data.py

```diff
@@ -646,11 +665,8 @@ def import_data(self, filename=None, selection=None):
             selection = []
         library = self._initialize_library(filename)
 
-        controls = []
         if selection:
-            for thing in selection:
-                maya_lib.core.has_shape_of_type(thing, 'nurbsCurve')
```

Comment:
Was this lookup intentional or intended to be part of an if statement originally? @louisVottero

Reply:
Marking as resolved, as current code changes handle the check.